### PR TITLE
fix: at_commons remove isPaginated check in sync verb builder 

### PR DESCRIPTION
**Pull request by murali-shris**
**- What I did**
- remove isPaginated check in sync verb builder so limit: is always sent since server expects this value in sync:from verb
**- How I did it**
- removed isPaginated logic in SyncVerbBuilder
- deprecated sync: verb since sync:from is used on server
**- How to verify it**
- AtClient tests should pass 
 https://github.com/atsign-foundation/at_client_sdk/pull/1450

### CODE REVIEW

**Review by srieteja — DISMISSED** [review]

**Review by gkc — CHANGES_REQUESTED** [review]
@murali-shris I know in [the other comment](https://github.com/atsign-foundation/at_client_sdk/pull/1448#discussion_r1879803959) I said `a PR in at_lookup to change SyncVerbBuilder` but I meant `a PR in at_commons` since SyncVerbBuilder is in at_commons

**Comment by murali-shris** [comment]
> @murali-shris I know in [the other comment](https://github.com/atsign-foundation/at_client_sdk/pull/1448#discussion_r1879803959) I said `a PR in at_lookup to change SyncVerbBuilder` but I meant `a PR in at_commons` since SyncVerbBuilder is in at_commons

Completed the changes.

File: packages/at_commons/lib/src/verb/sync_verb_builder.dart [hunk 1]
```diff
@@ -21,13 +21,10 @@ class SyncVerbBuilder implements VerbBuilder {
...
-    if (isPaginated) {
-      serverCommandBuffer.write('from:');
-    }
+    // removed isPaginated logic, since sync:from is the default sync verb on server
+    serverCommandBuffer.write('from:');
     serverCommandBuffer.write('$commitId');
-    if (isPaginated) {
-      serverCommandBuffer.write(':limit:$limit');
-    }
+    serverCommandBuffer.write(':limit:$limit');
```
Comment (gkc) [inline packages/at_commons/lib/src/verb/sync_verb_builder.dart:26]: Does the comment on line 15 mean that `limit` is ignored on the atServer? If so, then perhaps we shouldn't deprecate `limit` but instead respect it on the atServer side?

**Review by gkc — CHANGES_REQUESTED** [review]

File: packages/at_commons/lib/src/verb/sync_verb_builder.dart (hunk 1 above)
Comment (murali-shris) [reply packages/at_commons/lib/src/verb/sync_verb_builder.dart:26]: in SyncProgressiveVerbHandler, only fromCommitId and regex are read from verb params. limit is not read.
There are two options
1) rely on default limit - 25 that is configured in SyncProgressiveVerbHandler --> commitLog.getEntries and deprecate limit param in sync verb builder
2) Keep limit param in sync verb builder and modify SyncProgressiveVerbHandler to read limit from verb params and pass the limit to commitLog.getEntries

Comment (gkc) [reply packages/at_commons/lib/src/verb/sync_verb_builder.dart:26]: @murali-shris Option 2 please, and change the default in the SyncVerbBuilder to be in line with whatever the current default is on the server (25)

Comment (murali-shris) [reply packages/at_commons/lib/src/verb/sync_verb_builder.dart:26]: Done.
Server PR:
https://github.com/atsign-foundation/at_server/pull/2182
Client PR:
https://github.com/atsign-foundation/at_client_sdk/pull/1450

File: packages/at_commons/lib/src/verb/sync_verb_builder.dart [hunk 2]
```diff
@@ -21,13 +20,10 @@ class SyncVerbBuilder implements VerbBuilder {
   @override
   String buildCommand() {
     StringBuffer serverCommandBuffer = StringBuffer('sync:');
-    if (isPaginated) {
-      serverCommandBuffer.write('from:');
-    }
+    // removed isPaginated logic, since sync:from is the default sync verb on server
```
Comment (gkc) [inline packages/at_commons/lib/src/verb/sync_verb_builder.dart:23]: Don't need this comment in the code

**Review by gkc — APPROVED** [review]

**Pull request merged**